api/models/event.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+
+const bcrypt = require('bcrypt');
+const bcrypt_p = require('bcrypt-promise');
```

Comment:
non-model-related variables should be in `camelCase`

Reply:
ok, also I need to pass them to the user model